ui/appui-react/package.json

```diff
@@ -117,13 +117,18 @@
   "dependencies": {
     "@itwin/itwinui-icons-react": "^2.8.0",
     "@itwin/itwinui-illustrations-react": "^2.0.1",
+    "@stratakit/bricks": "catalog:",
+    "@stratakit/foundations": "catalog:",
```

Comment:
`@stratakit/foundations` should be an ?optional? peer dependency?
I'd like to be able to update its range w/o requiring a major version update of AppUI as well @mayank99

Reply:
See: https://github.com/iTwin/iTwinUI/blob/1c11beaabf68e41924bfba14f43959285c97af23/packages/itwinui-react/package.json#L136